subitem.go

```diff
@@ -21,21 +27,20 @@ type SubscriptionItemPriceDataParams struct {
 // For more details see https://stripe.com/docs/api#create_subscription_item and https://stripe.com/docs/api#update_subscription_item.
 type SubscriptionItemParams struct {
 	Params            `form:"*"`
-	ID                *string                                  `form:"-"` // Handled in URL
 	BillingThresholds *SubscriptionItemBillingThresholdsParams `form:"billing_thresholds"`
 	ClearUsage        *bool                                    `form:"clear_usage"`
+	OffSession        *bool                                    `form:"off_session"`
```

Comment:
Do we want to keep the comment for what is only supported on updates? I think it was useful

Reply:
I don't think these comments exist consistently, and rather than preserve them piecemeal I'd rather re-introduce them wholesale in a single commit once codegen is ~complete.

Reply:
I would recommend having those stay now since it will limit the diff in future codegen instead of having to do it twice and it's quite specific to codegen here. The earliest you decide how to handle the diff between create and update the more likely you are to find potential issues.
I won't block since it doesn't change functionality and no one can really see those unless they read the code, it's just a nudge to think about doing it early

Reply:
I've added the comment back but "inline" as

```go
OffSession *bool `form:"off_session"` // Only supported on update
```

Again quite willing to revisit later and segregate properties by method, but for now my focus is on extending codegen coverage as quickly and with as small diffs as possible.